pkg/api/universal/scheduler.go

```diff
@@ -19,15 +19,32 @@ import (
 
 	"google.golang.org/protobuf/types/known/emptypb"
 
+	apierrors "github.com/dapr/dapr/pkg/api/errors"
 	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
 	schedulerv1pb "github.com/dapr/dapr/pkg/proto/scheduler/v1"
 )
 
 func (a *Universal) ScheduleJob(ctx context.Context, inReq *runtimev1pb.ScheduleJobRequest) (*emptypb.Empty, error) {
-	// validate job details, date, schedule, etc??
-	var err error
-
 	metadata := map[string]string{"app_id": a.AppID()}
```

Comment:
do we want to standardize on `app_id` or `appId`?

Reply:
I've been using `app_id` everywhere for scheduler bc I dont see a standard

Reply:
app_id is the one in metrics. If you see all the metadata we have in Dapr, I see camelCase.

Reply:
So `appId `and not `appID`, ok that is fine. I can update the scheduler code to use `appId` everywhere, but would prefer to do so in a follow up PR, as the diff would be cleaner. lmk if you would prefer me to update it here instead

Reply:
If it was my decision I'd prefer `appID` :slightly_smiling_face: 

qq: why is the app ID part of the request metadata rather than a first class field? @cicoyle

Reply:
I guess it could be a first class field, but is really only needed for the jobName in etcd for the listing functionality per appID, so I passed it along in the metadata. that can change no problem

Reply:
I'd also prefer `appID`, well, I'd prefer `app_id` but believe `appID` is better than `appId`. Idk if that is something that warrants a discussion for standardizing on metadata in Dapr? Ik there is already a diff with metrics bc that uses `app_id`

Reply:
for now, I'll make this specific to errors and make the `metadata` -> `errMetadata` bc its not being consumed in the scheduler apis beyond actor related things